RelevantUseCases/financial_services.tex

```diff
@@ -21,7 +21,7 @@ \subsection{Post trade activities overview}
 \textbf{segue why blockchain}
 
 \subsection{Efficiency benefits}
-Implementing post-trade-processing on blockchain is bound to lead to efficiency gains as compared to the current implementation model. When settling via a blockchain system one could exploit the peer-to-peer property of a blockchain, i. e. one counterparty would insert the transaction details into the system and the other counterparty would verify and confirm.  Thus the confirmation processes would be processed within the same system, rendering separate confirmation processes obsolete. In today’s world both parties would independently send their settlement instructions to a trusted 3rd party – the settlement agent – and this 3rd party would match both data sets and further process the settlement. Any mismatches in the initial instructions would lead to reconciliation efforts or even failed trades.
+Implementing post-trade-processing on blockchain is bound to lead to process efficiency gains as compared to the current implementation model. When settling via a blockchain system one could exploit the peer-to-peer property of a blockchain, i. e. one counterparty would insert the transaction details into the system and the other counterparty would verify and confirm.  Thus the confirmation processes would be processed within the same system, rendering separate confirmation processes obsolete. In today’s world both parties would independently send their settlement instructions to a trusted 3rd party – the settlement agent – and this 3rd party would match both data sets and further process the settlement. Any mismatches in the initial instructions would lead to reconciliation efforts or even failed trades.
```

Comment:
The last two sentences are not really needed IMO, but non-financial folks should weigh in.